Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/ViewBookmarkComponent.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzToolsFramework/ToolsComponents/EditorComponentBase.h>
+
+namespace AzToolsFramework
+{
+    struct ViewBookmark
+    {
+        AZ_CLASS_ALLOCATOR(ViewBookmark, AZ::SystemAllocator, 0);
+        AZ_RTTI(ViewBookmark, "{522A38D9-6FFF-4B96-BECF-B4D0F7ABCD25}");
+
+        ViewBookmark() = default;
+
+        static void Reflect(AZ::ReflectContext* context);
+
+        AZ::Vector3 m_position = AZ::Vector3::CreateZero();
+        AZ::Vector3 m_rotation = AZ::Vector3::CreateZero();
+    };
+
+    struct EditorViewBookmarks final
+    {
+        AZ_CLASS_ALLOCATOR(EditorViewBookmarks, AZ::SystemAllocator, 0);
+        AZ_RTTI(EditorViewBookmarks, "{EA0B8FF9-F706-4115-8226-E3F54F1EE8A1}");
```

Comment:
This class shouldn't use AZ_RTTI since it is final and doesn't inherit from any class.
It should use `AZ_TYPE_INFO`